README.md

```diff
@@ -132,6 +132,7 @@ The Unleash SDK takes the following options:
 | customHeaders     | no| `{}` | Additional headers to use when making HTTP requests to the Unleash proxy. In case of name collisions with the default headers, the `customHeaders` value will be used if it is not `null` or `undefined`. `customHeaders` values that are `null` or `undefined` will be ignored. |
 | impressionDataAll | no| `false` | Allows you to trigger "impression" events for **all** `getToggle` and `getVariant` invocations. This is particularly useful for "disabled" feature toggles that are not visible to frontend SDKs. |
 | environment | no | `default` | Sets the `environment` option of the [Unleash context](https://docs.getunleash.io/reference/unleash-context). This does **not** affect the SDK's [Unleash environment](https://docs.getunleash.io/reference/environments). |
+| togglesStorageTTL   | no | `0` | How long (Time To Live), in seconds, the toggles in storage are considered valid and should not be fetched again. If set to 0 will disable expiration checking and will be considered always expired.                 |
```

Comment:
Is there any reason we would not default this to the `refreshInterval`? 

I can see the argument of having it "opt-in" initially, until we validate the behavior, but it would be great win for everyone to not overload the Unleash Frontend API (or Edge) if a user decides to force-reload multiple times?

Reply:
> Is there any reason we would not default this to the refreshInterval?

This is because in our case, we don't want to use the `refreshInterval` because it uses `setInterval` that we can't use in our "browser extension" context. Moreover if anyone doesn't want the refresh feature to be enabled, they will not be able to use this new feature if there is no option for it.

> I can see the argument of having it "opt-in" initially, until we validate the behavior, but it would be great win for everyone to not overload the Unleash Frontend API (or Edge) if a user decides to force-reload multiple times?

Do you mean this should be enabled by default?